content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -93,30 +93,30 @@ to interface with your chosen container runtime.
 
 If you don't specify a runtime, kubeadm automatically tries to detect an installed
 container runtime by scanning through a list of well known Unix domain sockets.
-The following table lists container runtimes and their associated socket paths:
+The following table lists container runtimes that the kubelet looks for, and their associated socket paths:
 
 {{< table caption = "Container runtimes and their socket paths" >}}
-| Runtime    | Path to Unix domain socket        |
-|------------|-----------------------------------|
-| Docker     | `/var/run/dockershim.sock`        |
-| containerd | `/run/containerd/containerd.sock` |
-| CRI-O      | `/var/run/crio/crio.sock`         |
+| Runtime        | Path to Unix domain socket        |
+|----------------|-----------------------------------|
+| Docker Engine  | `/var/run/dockershim.sock`        |
+| containerd     | `/run/containerd/containerd.sock` |
+| CRI-O          | `/var/run/crio/crio.sock`         |
 {{< /table >}}
 
 <br />
-If both Docker and containerd are detected, Docker takes precedence. This is
+If both Docker Engine and containerd are detected, the kubelet gives precedence to Docker Engine. This is
```

Comment:
```suggestion
If both Docker Engine and containerd are detected, kubeadm will give precedence to Docker Engine. This is
```

kubeadm is the component doing the detection (in < 1.24)

or use:
> If both Docker Engine and containerd are detected, Docker Engine takes precedence. This is

Reply:
Ah, yes. I knew it was something that began with `k`.